dashboard/app/controllers/levels_controller.rb

```diff
@@ -132,7 +132,7 @@ def show
 
     view_options(
       full_width: true,
-      small_footer: @game.uses_small_footer? || @level.enable_scrolling?,
+      small_footer: @game&.uses_small_footer? || @level&.enable_scrolling?,
```

Comment:
this also applies to line 535 in `dashboard/app/controllers/script_levels_controller.rb`, but should a fallback for the OR operator be provided (i.e. `|| false`)? if `@level` has the potential of not including the enable_scrolling method, then a fallback should be provided